proto/protovalidate-testing/buf/validate/conformance/cases/groups_editions.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023-2025 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+edition = "2023";
+
+package buf.validate.conformance.cases;
+
+import "buf/validate/validate.proto";
+
+message GroupDelimited {
```

Comment:
Worth expanding this into the same cases as below? A repeated field and also one with presence == legacy_required?

Reply:
We already have legacy required stuff captured in the other editions cases iirc, and since this really only impacts the wire format it's I feel sufficient for now alongside the proto2 tests.